modules/module_http_test.go

```diff
@@ -0,0 +1,32 @@
+//go:build linux
+// +build linux
+
+package modules
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestHttpCheck(t *testing.T) {
+
+	h := &HttpModule{}
```

Comment:
This is spelled incorrectly - It should be `HTTPModule` - as per the status-check that ran against the pull-request.